src/modules/form/components/group/FormCard.tsx

```diff
@@ -140,6 +154,12 @@ const FormCard: React.FC<Props> = ({
           </Stack>
         )}
 
+        {item.helperText && (
```

Comment:
I'd like to ask for feedback/pointers about accessibility for the signature text inputs. On Friday we had discussed using `fieldset` and `legend`, but I'm a bit unsure about how to fit this in with the generic `TextInput` component. I thought about using `helperText` for the aria label, but `helperText` is already in use for the grey label beneath the text input box, like this:
<img width="612" alt="Screenshot 2024-02-13 at 10 21 31 AM" src="https://github.com/greenriver/hmis-frontend/assets/16002775/c96d6fd1-7d21-4e1a-b8f4-bd3614b4c678">

What do you suggest? Should we add a new form field attribute `accessibilityText`? or re-use `helperText`? or something else?

Reply:
Good question,  I'm didn't mean to suggest that the helperText on the group should be used on the input components themselves, sorry for the confusion. I was imagining that a fieldset would be a good way to improve accessibility of the inner group component using something like the following structure:
```jsx
<fieldset>
  <legend>Head of household</legend>
  <TextInput label="Full Name">
  <TextInput label="Today's date">
</fieldset>
```
So the group component would render fieldset/legend elements instead of divs/h3s. A screen reader should then be able to describe that the focused "Full Name" is for head of house household and distinguish it from the case manager full-name below.

I don't know if helper text within a card needs special treatment itself. A simple p tag might be enough; could test it it in a screen reader and add aria-describedby if needed. Looking at our current current implementation, the encapsulating element for these cards probably should be a `<section/>` rather than a div. This might be scope creep though...

Happy to discuss on slack

Reply:
Thanks! Let me know if this is along the lines of what you had in mind?

Reply:
Yes, this looks great to me, thank you!!